app/client/components/helpCentre/helpCentreSingleTopic.tsx

```diff
@@ -0,0 +1,52 @@
+import { css } from "@emotion/core";
+import React from "react";
+import { trackEvent } from "../analytics";
+import { HelpCentreTopic } from "./helpCentreConfig";
+import {
+  linkAnchorStyle,
+  linkArrowStyle,
+  linkListItemStyle,
+  linksListStyle
+} from "./helpCentreStyles";
+
+interface HelpCentreSingleTopicProps {
+  selectedHelpCentreConfigTopic: HelpCentreTopic;
```

Comment:
This components doesn't have knowledge that there are several topics elsewhere. It merely render what is passed. Neither does it need to have context that the passed topic comes from a config. For those reasons I suggest renaming it to `topic`.